lib/spack/spack/bootstrap.py

```diff
@@ -97,48 +99,139 @@ def _try_import_from_store(module, abstract_spec_str):
     return False
 
 
+def _executables_in_path(executables):
+    """Return True if at least one of the executables is in PATH,
+    False otherwise.
+    """
+    msg = "[BOOTSTRAP EXECUTABLES {0}] Look if the executables are in the path"
+    executables_str = ', '.join(executables)
+    tty.debug(msg.format(executables_str))
+    found = spack.util.executable.which_string(*executables)
+    if found:
+        msg = "[BOOTSTRAP EXECUTABLES {0}] {1} executable found in PATH"
+        tty.debug(msg.format(executables_str, found))
+        return True
+    return False
```

Comment:
Couldn't this method just be replaced with `bool(spack.util.executable.which_string(*executables)`? I think we're getting a little overly verbose, even for debug mode with some of this.